misc/schema/xrc_schema.rnc

```diff
@@ -760,7 +760,8 @@ wxCheckBox =
         stdObjectNodeAttributes &
         stdWindowProperties &
         [xrc:p="important"] element label {_, t_text }* &
-        [xrc:p="o"] element checked {_, t_bool }*
+        [xrc:p="o"] element checked {_, t_bool }* &
+        [xrc:p="o"] element undetermined {_, t_bool }*
```

Comment:
I wonder if we should be really using `bool` for this one. This allows using `<undetermined>false</undetermined>` which is rather strange. Perhaps we should extend `checked` to allow it taking some other value? The trouble is that I don't know which value would fit with the currently allowed `0` and `1` values, neither `-1` nor `2` seem like a good fit.

Reply:
We need to take some decision here if we want to have this in 3.1.7, so let's use 0, 1 and 2 for the states if nobody objects too much.

TIA!